model/result/messages/LisOutcomeRequestParser.php

```diff
@@ -134,10 +132,7 @@ protected function getOperationNode(DOMXPath $xpath)
         return $nodes->item(0);
     }
 
-    /**
-     * @return OperationsCollection
-     */
-    protected function getOperationsCollection()
+    protected function getOperationsCollection(): OperationsCollection
```

Comment:
Just to validate, as I saw many good refactoring here :) Please, be sure the protected methods are not used in somewhere with different signature / type hint / return type.